docs/user/guides/administrators.rst

```diff
@@ -13,6 +13,7 @@ have a look at our :doc:`/tutorial/index`.
    :maxdepth: 1
 
    technical-docs-seo-guide
+   Use Traffic Analytics </analytics>
```

Comment:
This is an odd title, and again we should just use the article title directly?

Reply:
See comment above.